src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseViewField.java

```diff
@@ -121,4 +122,10 @@ public Boolean getShowSummaryChangeOption() {
     public void setShowSummaryChangeOption(final Boolean showSummaryChangeOption) {
         this.showSummaryChangeOption = showSummaryChangeOption;
     }
+
+    @Override
+    public String toString() {
+        return ReflectionToStringBuilder.toString(this);
```

Comment:
beware of using reflection. Can slow down performance. Perhaps an IDE generated toString() better?

Reply:
Yeah maybe we should agree on a standard here. I can see many different usages across the code base including:
- StringBuilder,
- ReflectionToStringBuilder,
- commons-lang ToStringBuilder,
- commons-lang3 ToStringBuilder

I would lean towards the last option. What are your thoughts?

Reply:
ok changed to commons-lang3 ToStringBuilder